isic_archive/tasks/zip.py

```diff
@@ -157,6 +157,10 @@ def ingestBatchFromZipfile(self, batchId):
                         logger.exception('An individual image failed to be extracted')
                         continue
 
+                    file_ = File().load(resp.json()['_id'], force=True)
```

Comment:
`originalFile` would be more helpful.

Reply:
This is the one place where that won't work because `imageRole` isn't set yet.

Reply:
Sorry, I was unclear. I just meant to change the name of this local variable.